files/galaxy/tpv/destinations.yml.j2

```diff
@@ -279,6 +279,18 @@ destinations:
       require:
         - cz-pulsar
 
+  pulsar_egi01_tpv:
+    inherits: pulsar_default
+    runner: pulsar_eu_egi01
+    max_accepted_cores: 8
+    max_accepted_mem: 15
```

Comment:
At the moment we have a virtual HTCondor cluster with 6 compute nodes, 8 vCPUs and 16 GB each.

In the `max_accepted_cores` and `max_accepted_mem` do you include the total per node or per cluster?

Reply:
max_accepted_cores: Number of cores of the machine with the most cores in your cluster (8)
max_accepted_mem: Number of cores of the machine with the most memory in your cluster (15 looks good to me because you are going to need some memory for the OS, etc)

Reply:
thanks, then it looks correct